HACKME.md

```diff
@@ -0,0 +1,15 @@
+# Wetware Developer's Guide
```

Comment:
This is awesome.  I'm glad we have this! 👍

Reply:
You did this ahahaha